docs/source/migration/4.0.mdx

```diff
@@ -5,31 +5,82 @@ description: From 3.x
 
 <Note>
 
-**Version 4 is currently under development and this page itself is a work in progress.**
+**Version 4.0 is currently under development and this page itself is a work in progress.**
```

Comment:
I think I'd stick with `Apollo Kotlin 4` and `Apollo Kotlin 3`. It's generic and will age better. 

In places where we don't want to repeat ourselves, I'd go with `3.x` and `4.x`. I don't see a strong reason to make a distinction between `3.0` and `3.x`? (or `4.0` and `4.x`?)

We don't technically have a `4.0` release. I guess it's ok in the url because we've always done that but would otherwise avoid it.